src/main/java/com/lorepo/icplayer/client/module/text/WCAGUtils.java

```diff
@@ -178,6 +176,17 @@ private static String updateLinks(String text) {
 		return text.replaceAll("<a[^>]*?href=\"(.*?)\"*?>(.*?)</a>", LINK_START + " $2 " + LINK_END);
 	}
 
+	public static String removeSpaceAfterAltTextDot(String text) {
+		// Chrome implementation of TTS reads the dot if a space is present after it
+		return text.replaceAll("\\\\alt\\{ \\|\\. \\}</li>", "\\\\alt{ |.}</li>");
```

Comment:
Twój regex sprawdzić się tylko dla sytuacji ze spacją.
Jednak nie sprawdzi się z np. Twardą spacją lub innym białym znakiem.
Twarda spacja jest istotnym przypadkiem, gdyż klient po stworzeniu lekcji nie odróżni twardej spacji od zwykłej spacji.
Spójrz na: https://linuxhint.com/use-regex-whitespace-in-java/